app/src/main/java/com/github/sdpteam15/polyevents/database/DatabaseInterface.kt

```diff
@@ -363,4 +361,6 @@ interface DatabaseInterface {
         uid: String = (User.currentUser as UserInterface).uid,
         userAccess: UserInterface = User.currentUser as UserInterface
     ): Observable<Boolean>
+
+    fun getAvailableItems(): Map<String, Int>
```

Comment:
It would be necessary to add asynchronous and a description for this method.

Reply:
I will add the documentation yes. For the asynchronous, it might be for later because this function is actually used in the FakeDatabase to provide stubs data for the demo.

Reply:
Absolutely, I will add a comment for this function.